App.tsx

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 import Header from './src/components/header/header';
+import * as Font from 'expo-font';
 
 export default function App() {
+  //componentDidMount() {
+    //Font.loadAsync({
+      //'open-sans-bold': require('./assets/fonts/OpenSans-Bold.ttf'),
```

Comment:
All commented-out code should be deleted, unless there's a suuuper good reason for it.